### DISCUSSION TITLE
CWG Weekly Meeting | Week 3 | Agenda

### DISCUSSION THREAD

**Discussion by ravi-prakash-v** [beckn/protocol-specifications#280] [Announcements]
Looking for Agenda points for Core Working Group meeting on Week 03, 2022

**Comment by ravi-prakash-v** [comment]
- Discussions to be closed and submitted as proposals
- Proposals to be moved to Protocol Draft
- Protocol Drafts to be moved to Protocol Standards

**Comment by pkapustin** [comment]
I would like to suggest a couple of issues to discuss:
- Design principles to allow interoperability: precise types with no ambiguity or redundancy.
- HTTP adaptation of the core spec.
